### DISCUSSION TITLE
UTM running Windows 11 update causes screen flash and 'no toolbar', just like for everyone else...

### DISCUSSION THREAD

**Discussion by daniel-poirot** [utmapp/UTM#5328] [Windows]
TL;DR - How do I 'safe boot' in UTM?

The latest Windows Update has caused my ARM64 Tiny11 Windows (with my license) (on Mac M2) to flash or reset the display explorer. I am running the `virtio-ramfb` driver. (The other non-gl drivers don't initialize.)

This is common for native hardware, or other VM systems: 
https://www.minitool.com/data-recovery/fix-windows-11-flashing-screen-no-taskbar.html

The remediation is to run DISM and SFC. I can't get to a console to do that.

How do I boot into 'safe mode' running UTM?

**Comment by ZeekWorld** [comment]
I am hitting this same issue. On M2, Windows 11 keeps flickering and Menu bar is missing.
